.canon.yaml

```diff
@@ -0,0 +1,8 @@
+viam-orb-slam3:
+    image_amd64: ghcr.io/viamrobotics/canon:amd64
+    image_arm64: ghcr.io/viamrobotics/canon:arm64
```

Comment:
Just curious if there's a reason to use canon:arm64 here rather than canon:arm64-cache as is [done in rdk](https://github.com/viamrobotics/rdk/blob/cc7f18eecfccb321372b373f045500d0f6a8308d/.canon.yaml#L3-L4)?

Reply:
the -cache variants simply do a "make build" on the RDK repo, then include the contents of ~/go/ and ~/.cache/ in the final image. For non-RDK projects, most of that would just be wasted space. (But for RDK's CI, it saves several GB of download on most runs.)